.github/workflows/pr-build.yml

```diff
@@ -9,31 +9,55 @@ on:
 
 jobs:
   build:
+    if: ${{ github.actor != 'dependabot[bot]' }}
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout Code
```

Comment:
We should be testing using all supported node versions, but don't need to add it to this pr https://github.com/Availity/availity-workflow/blob/ef1e7cff67eebdb83fec924c1414ad5d21aa4538/.github/workflows/pr-build.yml#L13-L17